<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>transfercall</class>
    <widget class="QDialog" name="transfercall">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>302</width>
                <height>131</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="windowTitle">
            <string>Transfer</string>
        </property>
        <layout class="QVBoxLayout">
            <item>
                <widget class="QLineEdit" name="destinationNumberInput" />
            </item>
            <item>
                <layout class="QHBoxLayout">
                    <item>
                        <widget class="QPushButton" name="cancelButton">
                            <property name="text">
                                <string>Cancel</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="transferButton">
                            <property name="text">
                                <string>Transfer</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
